CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 6.0.0 - 2021-01-19
+
+* Add asynchronous code support
+* Change ``%s`` to f-strings
+
```

Comment:
No need to add this, we'll add it when we release it

Reply:
fixed